adminapi/src/main/java/io/minio/admin/UserInfo.java

```diff
@@ -93,7 +94,7 @@ public static Status fromString(String statusString) {
         return DISABLED;
       }
 
-      throw new IllegalArgumentException("Unknown status " + statusString);
+      return UNKNOWN;
```

Comment:
There is no unknown state from MinIO server. It is better to throw an error so that we can address actual value instead of treating it as `Unknown`